get-to-know-security-manager/microsoft-teams-screen.md

```diff
@@ -13,18 +13,42 @@ To start exploring features and reports, navigate to the Microsoft Teams screen
 
 Available management actions for Microsoft Teams are contextually related to the selected object, which means users can click on or hover over something to get a contextual menu that offers actions related to the selected item.
 
-## Available actions
+## All Teams report
+### All Teams Tab
+General overview of the Teams on your tenant. The report will show you the team's Display Name, privacy level, site URL, number of: channels, team members, owners, guests and the date when the team was created. 
```

Comment:
možda bi počeo rečenicu sa 'Gives a general overview..'
zarez ispred 'and the date when...'

Kopiram iz grammarlya:
Gives a general overview of the Teams on your tenant. The report will show you the team's Display Name, privacy level, site URL, number of channels, team members, owners and guests, and the date when the team was created.